easybuild/easyconfigs/g/GAMESS-US/GAMESS-US-20240715-R2_fix_rungms.patch

```diff
@@ -0,0 +1,67 @@
+diff -ruN gamess.orig/rungms-dev gamess/rungms-dev
```

Comment:
@sven-hansen please add description + author or reference to upstream sources on top of all patches